packages/cli/lib/sandboxes.js

```diff
@@ -1,73 +1,66 @@
-const chalk = require('chalk');
 const { i18n, MISSING_LANGUAGE_DATA_PREFIX } = require('./lang');
 const { handleExit, handleKeypress } = require('./process');
 const { logger } = require('@hubspot/cli-lib/logger');
-const {
-  getAccessToken,
-  updateConfigWithAccessToken,
-} = require('@hubspot/local-dev-lib/personalAccessKey');
 const { EXIT_CODES } = require('./enums/exitCodes');
-const { enterAccountNamePrompt } = require('./prompts/enterAccountNamePrompt');
 const {
   fetchTaskStatus,
   fetchTypes,
   getSandboxUsageLimits,
 } = require('@hubspot/local-dev-lib/sandboxes');
-const {
-  accountNameExistsInConfig,
-  getConfig,
-  writeConfig,
-  updateAccountConfig,
-  getAccountId,
-} = require('@hubspot/local-dev-lib/config');
+const { getConfig, getAccountId } = require('@hubspot/local-dev-lib/config');
 const CliProgressMultibarManager = require('./CliProgressMultibarManager');
 const { promptUser } = require('./prompts/promptUtils');
 const { getHubSpotWebsiteOrigin } = require('@hubspot/local-dev-lib/urls');
 const {
-  personalAccessKeyPrompt,
-} = require('./prompts/personalAccessKeyPrompt');
-const { logErrorInstance } = require('./errorHandlers/standardErrors');
-
-const STANDARD_SANDBOX = 'standard';
-const DEVELOPER_SANDBOX = 'developer';
+  HUBSPOT_ACCOUNT_TYPES,
+} = require('@hubspot/local-dev-lib/constants/config');
+const {
+  STANDARD_SANDBOX_TYPE,
```

Comment:
Are these different than what we have in `HUBSPOT_ACCOUNT_TYPES `? Is there any way we could get rid of the need for these constants in favor of the export from local-dev-lib?

Reply:
Yep - same as the other comment, I think I'll get around to swapping these once I get to the `sandboxAccountType` migration issue